server/command.go

```diff
@@ -69,6 +72,17 @@ func getAutocompleteData() *model.AutocompleteData {
 	embedded.AddStaticListArgument("Choose where the Jitsi meeting should open", true, items)
 	settings.AddCommand(embedded)
 
+	showPrejoinPage := model.NewAutocompleteData("show_prejoin_page", "[value]", "Choose wheather the pre-join page should be visible on Jitsi meeting in embedded mode")
+	items = []model.AutocompleteListItem{{
+		HelpText: "Pre-join page on Jitsi meeting will be displayed",
```

Comment:
Here i think its better to have different help text for these two options like 
```suggestion
		HelpText: "Show the pre-join page on Jitsi meeting will be displayed",
```